### PR TITLE
Responsive layout containers for home page, guide & story pages

### LINKED ISSUE #113: Define responsive layout container
## Questions

- How wide is widest with 3 columns? do we space between or outsides.
- What is the min & max on columns?
- TBC - dev to discuss with Emma
- This may differ on home & other pages (i.e. there may be 3 or more layouts)

## Plan

- [x] #134
- [x] Decide on some sensible breakpoints
- [x] Decide on some sensible margin sizes & padding
- [x] Create some generic css utility classes for things like padding (a clearfix-type block)
- [x] Create the required layout classes for use in default pages
- [ ] #143
- [ ] #144
- [ ] #145

## Linked task

- [ ] Update notion to add in some CSS snippets for useful utility vars

### PR DESCRIPTION
**Pull request by r-ferrier**
Fixes #113 

## Description

- Adds a generic set of responsive layout containers to provide sensible styling at mobile, tablet, & desktop
- Styles the home page, guide & story pages, and some shared elements like teaser images
- Styles have been placed in global as they are reused throughout
-  If an optional part of the design is not present (eg a list of related guides) then the design should fill the remaining space
- I've tried not to let any of the global classes I defined get so big that they can't be broken up and reused, but it's hard to know when that's helpful/ when that's a hindrance so if we want to break them down a bit more/ join some of them back together again let me know!

- I've also created a big layouts file which has been a really helpful reference while making this. I guess this might be helpful as visual acceptance criteria, to check that the layout looks as it should:

https://www.figma.com/file/rLewb9kn4mUt8lLI0wuC7f/Team-wilder-layouts?node-id=0-1&t=9wL7svPl8A1zfV88-0

## still to do

- The header & footer - will leave these while Ivan is making footer logos & doing the header h1 thingmy - think it would be better to get these in first - but have some layouts designed for them. Will make a new issue.
- The generic styling library we wanted to add to notion - will have a think about this next week in my in between times, once this is in

@geeksforsocialchange/developers

### CODE REVIEW

File: content/guides/en/types-of-community-group.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ summary: |-
...
+  - title: "Types of Community Group"
+    slug: types-of-community-group
+relatedStories:
+  - title: "[cCc] Dummy story"
+    slug: "story-slug"
+    image:
+      src: "/images/wildlife-trust-logo.png"
+      alt: "Wildlife Trusts logo"
+    description: "A short description of my story"
 ---
```
Comment (r-ferrier) [inline content/guides/en/types-of-community-group.md:20]: added for reference while testing

File: src/I18n/En.elm [hunk 2]
```diff
@@ -123,3 +123,6 @@ enStrings key =
 
         WelcomeP3 ->
             "[cCc] The goal of Nextdoor Nature is to get at least 1 in 4 people across the UK taking Action for Nature. This online library aims to support anyone who is interested in finding out what they can do, and how they can get other people involved."
+
+        ExploreGuidesListPlaceholder ->
+            "[cCc] Explore guides by theme"
```
Comment (r-ferrier) [inline src/I18n/En.elm:128]: added as a placeholder for the homepage to make visualising things easier

**Comment by ivan-kocienski-gfsc** [comment]
I think this is okay. It gives us the infrastructure to be able to expand and improve the layout breakpoints.

File: src/Theme/PageTemplate.elm [hunk 3]
```diff
@@ -28,7 +23,7 @@ view model content =
     div [ css [ pageWrapperStyle ] ]
         [ globalStyles
         , div [ css [ pageStyle ] ]
-            [ main_ [ css [ mainStyle ] ]
+            [ main_ []
                 [ button [ onClick LanguageChangeRequested ] [ text (t ChangeLanguage) ]
                 , div []
```
Comment (katjam) [inline src/Theme/PageTemplate.elm:28]: We might want to move the `pageWrapperStyle` `div` here and possibly the container inside that also if it is common to all pages.

**Review by katjam — APPROVED** [review]
This looks good. I've left a couple of comments, you can act on them if you like. We also need to make the cookie banner sticky to the bottom, but I'll add that to the AC for footer if you haven't already done so.

**Comment by r-ferrier** [comment]
I've made all the recommended changes and am merging so we can get on with the next bits of styling.

**Pull request merged**